rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -154,8 +154,8 @@ qubesctl saltutil.sync_all refresh=true -l quiet --out quiet > /dev/null || true
 qubesctl top.enable securedrop_salt.sd-workstation > /dev/null ||:
 
 # Force full run of all Salt states - uncomment in release branch
-# mkdir -p /tmp/sdw-migrations
-# touch /tmp/sdw-migrations/fedora-selinux-fix
+mkdir -p /tmp/sdw-migrations
+touch /tmp/sdw-migrations/undo-fedora-selinux-fix
```

Comment:
With the benefit of a bit more time for this revert: Do we in fact need to perform a full `sdw-admin --apply` run to apply this change? The updater does apply the `dom0` state in every run:
https://github.com/freedomofpress/securedrop-workstation/blob/34663f87f1b498932db3b957aa77384e293203c1/sdw_updater/Updater.py#L437

And the `sd-sys-vms.sls` is included in the `dom0` states in `sd-workstation.top`:
https://github.com/freedomofpress/securedrop-workstation/blob/34663f87f1b498932db3b957aa77384e293203c1/securedrop_salt/sd-workstation.top#L8